tests_e2e/lisa/testsuites/agent-bvt.py

```diff
@@ -0,0 +1,41 @@
+from assertpy import assert_that
+from pathlib import Path
+from tests_e2e.lisa.tests.agent_bvt import custom_script
+
+from lisa import (
+    CustomScriptBuilder,
+    Logger,
+    Node,
+    simple_requirement,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+)
+from lisa.sut_orchestrator.azure.common import get_node_context
+
+
+@TestSuiteMetadata(
+    area="bvt",
+    category="functional",
+    description="""
+    A POC test suite for the waagent BVTs.
+    """,
+    requirement=simple_requirement(unsupported_os=[]),
+)
+class AgentBvt(TestSuite):
+    @TestCaseMetadata(description="", priority=0)
+    def check_agent_version(self, node: Node, log: Logger) -> None:
+        script_path = CustomScriptBuilder(Path(__file__).parent.parent.joinpath("tests", "agent_bvt"), ["check_agent_version.py"])
+        script = node.tools[script_path]
+        result = script.run()
+        log.info(result.stdout)
+        log.error(result.stderr)
+        assert_that(result.exit_code).is_equal_to(0)
+
+    @TestCaseMetadata(description="", priority=0)
+    def custom_script(self, node: Node) -> None:
```

Comment:
Pattern 2: Execute a test locally (local to the orchestrator) to trigger extensions on the remote test VM

Reply:
At what point is the remote test VM created?

Reply:
LISA does that. In the output of the task you will see something similar to 

```
2022-10-25 15:13:49.326[140692391196416][INFO] lisa.[azure].deploy[generated_0] vm setting: AzureNodeSchema(name='lisa-azure-20221025-151340-053-e0-n0', short_name='lisa--053-e0-n0', vm_size='Standard_DS2_v2', maximize_capability=False, location='westus2', subscription_id='8e037ad4-****-****-****-********c15b', marketplace_raw={'publisher': 'canonical', 'offer': 'ubuntuserver', 'sku': '18.04-lts', 'version': '18.04.202210180'}, shared_gallery_raw=None, vhd='', hyperv_generation=1, purchase_plan=None, is_linux=True)
2022-10-25 15:13:53.044[140692391196416][INFO] lisa.[azure].deploy[generated_0] resource group 'lisa-azure-20221025-151340-053-e0' deployment is in progress...
2022-10-25 15:15:26.749[140692391196416][INFO] lisa.[azure].deploy[generated_0] deployed in 99.972 sec
2022-10-25 15:15:26.754[140692483270400][INFO] lisa.env[generated_0].node[0] initializing node 'lisa-azure-20221025-151340-053-e0-n0' w****t@20.230.174.145:22

```